test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... FAIL
test_to_degrees (__main__.TestExercice) ... FAIL
test_to_farenheit (__main__.TestExercice) ... FAIL
test_to_radians (__main__.TestExercice) ... ok

======================================================================
FAIL: test_to_celsius (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 95, in test_to_celsius
    self.assertListEqual(
AssertionError: Lists differ: [32.0, -13.0, 843.8000000000001] != [-17.77777777777778, -31.666666666666664, 232.77777777777777]

First differing element 0:
32.0
-17.77777777777778

- [32.0, -13.0, 843.8000000000001]
+ [-17.77777777777778, -31.666666666666664, 232.77777777777777] : Mauvais calcul de farenheit -> celsius

======================================================================
FAIL: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 83, in test_to_degrees
    self.assertListEqual(
AssertionError: Lists differ: [(0.0, 0.0, 0.0), (0.0, 0.0, 0.0), (0.0, 0.[20 chars]0.0)] != [(79.0980174562859, 5.881047377154118, 29.6[197 chars]998)]

First differing element 0:
(0.0, 0.0, 0.0)
(79.0980174562859, 5.881047377154118, 29.612494192724625)

- [(0.0, 0.0, 0.0), (0.0, 0.0, 0.0), (0.0, 0.0, 0.0), (0.0, 0.0, 0.0)]
+ [(79.0980174562859, 5.881047377154118, 29.612494192724625),
+  (-278.1960349125718, 48.23790524569176, 0.7750116145507491),
+  (257.74504364071475, -15.297381557114704, 44.031235481811564),
+  (213.95544569810457, -2.673258113725865, 23.057550624127998)] : Mauvais calcul de radians -> degres

======================================================================
FAIL: test_to_farenheit (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 107, in test_to_farenheit
    self.assertListEqual(
AssertionError: Lists differ: [-32.0, -45.888888888888886, 218.55555555555554] != [32.0, -13.0, 843.8000000000001]

First differing element 0:
-32.0
32.0

- [-32.0, -45.888888888888886, 218.55555555555554]
+ [32.0, -13.0, 843.8000000000001] : Mauvais calcul de celsius -> farenheit

----------------------------------------------------------------------
Ran 7 tests in 0.001s

FAILED (failures=3)
